public/javascripts/Admin/src/Admin.js

```diff
@@ -444,7 +606,7 @@ function Admin(_, $) {
                 }
                 var stats = getSummaryStats(data, "rate");
                 $("#neighborhood-std").html((stats.std).toFixed(2) + "%");
-
+                
```

Comment:
Make sure to double check your PR for random changes like this! We don't want this whitespace here and we don't want the empty comment below!

Reply:
Sorry about that I'll fix that asap.